src/argus_htmx/templates/htmx/base.html

```diff
@@ -12,7 +13,9 @@
     <header class="bg-neutral text-neutral-content">
         <nav class="navbar">
             <div class="navbar-start">
-                <a class="btn btn-ghost text-xl" href="{% url 'htmx:incident-list' %}">Argus Logo</a>
+                <a class="link" href="{% url 'htmx:incident-list' %}">
+                    <img class="object-scale-down h-14" src="{% static "logo_white.svg" %}" alt="Argus logo">
```

Comment:
I think just "Argus" or "Home" as the alt text is better.

Reply:
If we write "Argus" after the logo we could hide the logo completely from screen readers.